proto/prysm/v1alpha1/beacon_block.proto

```diff
@@ -76,15 +80,33 @@ message GenericBeaconBlock {
         // Representing a post-Capella fork blinded beacon block.
         BlindedBeaconBlockCapella blinded_capella = 6;
 
-        // Representing a signed, post-Deneb fork beacon block content.
+        // Representing a post-Deneb fork beacon block.
         BeaconBlockContentsDeneb deneb  = 7;
 
         BlindedBeaconBlockDeneb blinded_deneb  = 8;
+
+	// ePBS block
+	BeaconBlockePBS epbs = 9;
     }
     bool is_blinded = 100;
     string payload_value = 101;
 }
 
+// Eth1Data represents references to the Ethereum 1.x deposit contract.
+message Eth1Data {
+    // The 32 byte deposit tree root for the last deposit included in this
+    // block.
+    bytes deposit_root = 1 [(ethereum.eth.ext.ssz_size) = "32"];
+
+    // The total number of deposits included in the beacon chain since genesis
+    // including the deposits in this block.
+    uint64 deposit_count = 2;
+
+    // The 32 byte block hash of the Ethereum 1.x block considered for deposit
+    // inclusion.
+    bytes block_hash = 3 [(ethereum.eth.ext.ssz_size) = "32"];
+}
+
```

Comment:
Why did we move these from L271-L285?

Reply:
Sorry, tried to decouple the proto files and moved Eth1 data to another file,  but in the end the dependencies of slashings with headers make it impossible to have different files for protos on blocks. Desisted on finding the right place again :)